### ISSUE TITLE
NTR: ATPase-dependent chromatin remodeler activity

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#21609]
Please provide as much information as you can: 

* **Suggested term label:**
ATPase-dependent chromatin remodeler activity

* **Definition (free text)**
* Possible def from  (PMID: 14729263) "proteins that utilize the energy derived from ATP-hydrolysis to modulate the contacts between histones and DNA, resulting in a change in chromosome architecture within the nucleosomal array, a process commonly referred to as “chromatin remodeling” (Varga-Weisz and Becker, 1998).

* **Reference, in format PMID:#######**
PMID: 14729263

* **Parent term(s)**
* ?? 
* has_part DNA binding
* has_part ATP hydrolysis activity
* part_of chromatin remodeling

* **Children terms (if applicable)** Should any existing terms that should be moved underneath this new proposed term?

* **Synonyms (please specify, EXACT, BROAD, NARROW or RELATED)**

* **Cross-references**
* For enzymes, please provide RHEA and/or EC numbers. 
* Can also provide MetaCyc, KEGG, Wikipedia, and other links.

* **Any other information**

**Comment by pgaudet** [comment]
@colinlog

**Comment by sjm41** [comment]
Also: Snf2-family proteins: chromatin remodellers for any occasion (2011)
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4162295/

**Comment by pgaudet** [comment]
+[Term]
+id: GO:0140658
+name: ATPase-dependent chromatin remodeler activity
+namespace: molecular_function
+def: "An activity, driven by ATP hydrolysis, that modulate the contacts between histones and DNA, resulting in a change in chromosome architecture within the nucleosomal array, leading to chromatin remodeling." [PMID:14729263, PMID:21862382, PMID:30867599]
+is_a: GO:0008094 ! ATPase, acting on DNA
+relationship: has_part GO:0003677 ! DNA binding
+relationship: has_part GO:0016887 ! ATP hydrolysis activity
+relationship: part_of GO:0006338 ! chromatin remodeling
+property_value: term_tracker_item https://github.com/geneontology/go-ontology/issues/21609 xsd:anyURI
+created_by: pg
+creation_date: 2021-06-15T13:57:18Z

**Comment by bmeldal** [comment]
Spotted a typo :D :
Definition should read "...that **modulates**..." ("s" missing)

**Comment by sjm41** [comment]
In case of interest/relevance to this recent discussion - this new paper entered FlyBase just today:
https://pubmed.ncbi.nlm.nih.gov/34313222/

**In vivo analysis reveals that ATP-hydrolysis couples remodeling to SWI/SNF release from chromatin**

**Abstract**:
ATP-dependent chromatin remodelers control the accessibility of genomic DNA through nucleosome mobilization. However, the dynamics of genome exploration by remodelers, and the role of ATP hydrolysis in this process remain unclear. We used live-cell imaging of Drosophila polytene nuclei to monitor Brahma (BRM) remodeler interactions with its chromosomal targets. In parallel, we measured local chromatin condensation and its effect on BRM association. Surprisingly, only a small portion of BRM is bound to chromatin at any given time. BRM binds decondensed chromatin but is excluded from condensed chromatin, limiting its genomic search space. BRM-chromatin interactions are highly dynamic, whereas histone-exchange is limited and much slower. Intriguingly, loss of ATP hydrolysis enhanced chromatin retention and clustering of BRM, which was associated with reduced histone turnover. Thus, ATP hydrolysis couples nucleosome remodeling to remodeler release, driving a continuous transient probing of the genome.
